pipeline.d/service.yml

```diff
@@ -0,0 +1,17 @@
+---
+service:
+  family:
+    - core
+    - webui
+  name: developercenter-cdn--streaming-client-webui
```

Comment:
This seems to match the naming we have right now (and I found out that `/` isn't allowed in this field), but I feel a little wary. 🙂